code/datums/supplypacks/engineering.dm

```diff
@@ -73,6 +73,12 @@
 	containername = "emitter crate"
 	access = access_engine_equip
 
+/decl/hierarchy/supply_pack/engineering/rcdmatter
+	name = "Equipment - Matter Cartridges"
+	contains = list(/obj/item/rcd_ammo = 5)
+	cost = 10
+	containername = "matter cartridge container"
```

Comment:
You can print RCD cartridges with autolathe/protolathe.

Reply:
what if you dont have resources :trollface: